site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+---
+layout: doc
+title: Configure WebAuthn
+description: Learn how to configure WebAuthn for FusionAuth
+navcategory: admin
+---
+
+include::docs/v1/tech/shared/_advanced-edition-blurb.adoc[]
+
+== Overview
+
+WebAuthn provides the ability for users to authenticate in their browser using the same method they use to unlock their device, like a biometric scan or PIN. WebAuthn has some big security benefits over the traditional username and password. Read on to learn how to configure WebAuthn in your FusionAuth instance.
+
+* <<What is WebAuthn>>
+* <<Configuring WebAuthn>>
+** <<Tenant Configuration>>
```

Comment:
This is a style thing and I don't know if we're entirely consistent, but I'd make Tenant and Application Configuration Top Level (so `==`).